static/js/ducks/recentGcnEvents.js

```diff
@@ -0,0 +1,32 @@
+import messageHandler from "baselayer/MessageHandler";
+
+import * as API from "../API";
+import store from "../store";
+
+export const FETCH_RECENT_GCNEVENTS = "skyportal/FETCH_RECENT_GCNEVENTS";
+export const FETCH_RECENT_GCNEVENTS_OK = "skyportal/FETCH_RECENT_GCNEVENTS_OK";
+
+export const fetchRecentGcnEvents = () =>
```

Comment:
```suggestion
export const fetchGcnEvents = () =>
```
b/c there's no time filtering -- you're just fetching all of them.

Reply:
then this same action creator can be used for both the widget and the actual events page